lib/components/SpanTooltip/SpanTooltip.stories.tsx

```diff
@@ -1,18 +1,42 @@
-import type { Meta, StoryObj } from 'storybook-solidjs'
-import { default as SpanTooltipComponent } from './SpanTooltip'
 import React from 'react'
+import { Meta, StoryFn } from '@storybook/react'
+import SpanTooltip from './SpanTooltip'
+import type { SpanTooltipProps } from './SpanTooltip'
 
-const meta: Meta<typeof SpanTooltipComponent> = {
-  component: SpanTooltipComponent,
-  title: 'Components/SpanTooltip'
-}
+export default {
+  title: 'Components/SpanTooltip',
+  component: SpanTooltip,
+  argTypes: {
+    children: {
+      control: 'text',
+      description: 'Text or number to display inside the tooltip span.'
+    },
+    extraClasses: {
+      control: 'text',
+      description: 'Additional CSS classes for custom styling.'
+    },
+    style: {
+      control: 'object',
+      description: 'Inline styles for the tooltip span.'
+    },
+    additionalData: {
```

Comment:
`isMultiLine`?

Reply:
Added description

Reply:
I mean, do we still need the `additionalData` param, if `isMultiLine` does the trick?

Reply:
It is not exactly the same, 

1. `AdditionalData` appears always, also if we are not truncating 
2. You can add some additional data if you really need
3. We need to keep BWC, maybe it was used somewhere?

Reply:
1&2 - got it, makes sense, thanks
3 - what's BWC?:)

Reply:
Backward Compatibility :)